src/test/java/com/softserveinc/ita/FilterModalTest.java

```diff
@@ -0,0 +1,44 @@
+package com.softserveinc.ita;
+
+import com.softserveinc.ita.pageobjects.HomePage;
+import com.softserveinc.ita.pageobjects.ProductPage;
+import com.softserveinc.ita.utils.runners.TestRunner;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FilterModalTest extends TestRunner {
+
+    @DataProvider
+    public Object[][] priceSortingFunctionality() {
+        return new Object[][]{
+                {"Ноутбуки", "2 999", "8 999", "9 999"}};
+    }
+
+    @Test(dataProvider = "priceSortingFunctionality")
+    public void verifyPriceSortingFunctionality(String categoryName, String minPrice, String maxPrice, String invalidValue) {
+        var homePage = new HomePage();
+        homePage
+                .closeAdvertisingBannerIfDisplayed()
+                .selectRequiredCategory(categoryName)
+                .selectRandomSubCategory();
+
+        var productPage = new ProductPage();
+        productPage
+                .setMinimalPrice(minPrice.replaceAll("\\s", ""))
+                .setMaximalPrice(maxPrice.replaceAll("\\s", ""))
+                .clickOnPriceButton();
+
+        productPage.selectFromCheapToExpensive();
+        var getPriceFromFilteredItems = productPage.getPriceFromFirstItem();
+        assertThat(getPriceFromFilteredItems)
+                .as("Test failed: Minimal price should be " + minPrice)
+                .isGreaterThanOrEqualTo(minPrice);
+
+        productPage.selectFromExpensiveToCheap();
+        assertThat(getPriceFromFilteredItems)
```

Comment:
in this variable saved price from first item sorted by cheap to expensive, is it ok here

Reply:
its not because after first assert, im changing it by option 2, so my value will from 8999 to 2999, i mean expensive to cheap